libcextract/SymbolExternalizer.cpp

```diff
@@ -768,6 +754,131 @@ SymbolUpdateStatus *SymbolExternalizer::getSymbolsUpdateStatus(const StringRef &
   return &ret->second;
 }
 
+void SymbolExternalizer::Dump_SymbolsMap(void)
+{
+  SourceManager &sm = AST->getSourceManager();
+
+  for (auto it = SymbolsMap.begin(); it != SymbolsMap.end(); ++it) {
+    SymbolUpdateStatus &sym = it->getValue();
+    sym.Dump(sm);
+  }
+}
+
+void SymbolExternalizer::Compute_SymbolsMap_Late_Insert_Locations(std::vector<SymbolUpdateStatus *> &array)
```

Comment:
Maybe ``array`` as argument name could be improved? I mean, when looking at the code, the name ``array`` doesn't say much about what is containers.